docs/immortal-empires/norsca.md

```diff
@@ -0,0 +1,30 @@
+# Norsca
+
+## Wulfrik the Wanderer - World Walkers
+
+* **Route I - Ragnarok**
+  * The End Times are nigh. Wulfrik has long cursed the gods for his predicament, but now realises he must pledge himself to one of them if he is to survive the coming apocalypse.
+  * Attain level 3 allegiance with a Chaos God.
+   * Defeat all of the Challengers sent against you.
+  * **Reward**:
+	* _Seafang Empowered_: Campaign movement range +30% (factionwide), enables teleportation for Wulfrik from port cities he owns (possible?)
+_Reward flavour text:_ Seafang's figurehead quickly absorbed the blood of the slain challengers that were smeared upon it. The ship now glows with eldritch power like never before.
+
+* **Route II - The World Walker**
+  * After a drunken boast made in sight of the Dark Gods, Wulfrik was cursed to forever wander the earth and seek out the fiercest challengers and most monstrous creatures and slay them in single combat.
+  * Occupy at least 5 of the following settlements:
+	* Couronne, Miragliano, Karaz-a-Karak, Altdorf, Castle Drakenhof, Black Crag, Kislev, Naggarond, Lothern, Hexoatl, Skavenblight, Khemri, The Awakening

```

Comment:
Add WH3 settlements too here.

Reply:
We could but they have no researches currently (maybe this will change in a future Norsca patch) so it is pointless for Norsca to go for them at present.

Reply:
Do not think of it as a requirement but as a reward. You have to occupy at least 5 of a given list. That doesn't means you have to go to Cathay for any reason (and you are not encouraged to), but if you go, you get your reward. It is also more future proof because Norsca will surely get a rework at some point.

Reply:
Which settlements are to be added?

Reply:
Wei-Jin, Zhar-Naggrund, Fire Mouth/Maw Gate maybe.